sympy/core/tests/test_arit.py

```diff
@@ -1850,7 +1850,7 @@ def test_mul_coeff():
     # It is important that all Numbers be removed from the seq;
     # This can be tricky when powers combine to produce those numbers
     p = exp(I*pi/3)
-    assert p**2*x*p*y*p*x*p**2 == x**2*y
+    assert p**2*x*p*y*p*x*p**2 == x**2*y*(1/2 + sqrt(3)*I/2)**6
```

Comment:
This could probably be `(p**2*x*p*y*p*x*p**2).expand() == x**2*y` to avoid the extra factor,